.ci/jobs/bump-opbeans-node-mbp.yml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This PR contains both changes, the pipeline/scripts and the `jjbb`. I normally create two different PRs, the one for the `jjbb` that gets merged earlier, so I can iterate in the pipeline/scripts by having a pipeline in Jenkins that validates those changes. But since those are trivial, I'd say we are good to go